pkg/util/parametermap/map.go

```diff
@@ -0,0 +1,145 @@
+package parametermap
+
+import (
+	"fmt"
+	"strings"
+
+	"golang.org/x/exp/slices"
+	corev1 "k8s.io/api/core/v1"
+)
+
+type argument struct {
```

Comment:
is it something like Entry or Element? It's not clear what's argument, more like function argument for me.

Reply:
renamed.